packages/fuels-core/src/code_gen/functions_gen.rs

```diff
@@ -126,6 +116,38 @@ pub fn generate_script_main_function(
     })
 }
 
+pub fn generate_predicate_encode_function(
```

Comment:
Does this function need to be`pub`?

Reply:
Good catch. It needs to be `pub(crate)` only.